ExternalScripts(DONOTINCLUDEINRELEASE)/scraper/quest/quest_spider.py

```diff
@@ -17,14 +18,22 @@ def __init__(self) -> None:
         self.start_urls = [self.base_url_classic.format(quest_id) for quest_id in QUEST_IDS]
 
     def parse(self, response):
+        if response.url.startswith('https://www.wowhead.com/classic/quests?notFound='):
+            questID = re.search(r'https://www.wowhead.com/classic/quests\?notFound=(\d+)', response.url).group(1)
+            logging.warning('\x1b[31;20mQuest with ID {questID} not found\x1b[0m'.format(questID=questID))
+            return None
+
         result = {}
         for script in response.xpath('//script/text()').extract():
-            if script.startswith('//<![CDATA[\nWH.Gatherer.addData') and script.endswith('//]]>'):
+            if script.startswith('//<![CDATA[\nWH.Gatherer.addData') and script.find('$.extend(g_quests') >= 0 and script.endswith('//]]>'):
```

Comment:
This additional check ensures we look at the right script tag. Sometimes multiple script tags match the other two checks and then the script fails, because it cannot find the questId.